rfcs/convergence/authoring-stories.md

```diff
@@ -267,13 +271,144 @@ TBA
 
 TBA
 
-### 7. how structure stories in the storybook nav tree
+### 7. dissecting big story files into smaller ones
 
-TBA
+Components with complex API require many long stories, both for documentation and for e2e testing. This creates big story files, which are hard to maintain. How can these files be dissected into smaller ones?
 
-### 8. dissecting big story files into smaller ones
+#### Proposal
 
-TBA
+1. Every component can have at most one `.stories.tsx` file with default export which configures metadata about the component. This file must be called `Component.stories.tsx`, for example `Button.stories.tsx`.
+2. If putting all stories into one file would make it too long, individual stories might be put into additional `.stories.tsx` files as a named export, and then re-exported from `Component.stories.tsx` file like this: `export * from ‘./IndividualStoryFile.stories’;`
+3. If individual story files are employed, `Component.stories.tsx` file must not contain any stories besides the default export.
+
+
+##### Good Example 1 - single file
+
+Button.stories.tsx:
+```tsx
+import { Meta } from '@storybook/react';
```

Comment:
**suggestion** no need to add those as we need to properly document in another RFC way how we wanna type stories

Reply:
How about we keep it for now and remove it in the PR that’ll propose typing of stories? Just a safer way in my opinion, cause plans don’t always get done :laughing:

This is just proper usage of Storybook with Typescript according to [their docs](https://storybook.js.org/docs/react/writing-stories/introduction), so no new best practice/convention is being proposed here - it’s just a reminder.

Reply:
> How about we keep it for now and remove it in the PR that’ll propose typing of stories?

thats definitely works. please not that this comment type was a `suggestion` -> pls refer to following standard to understand type of comments in the future https://conventionalcomments.org/